language-design.md

```diff
@@ -26,15 +47,33 @@ you know a poorly designed language? What are the symptoms?
 
 **Response**
 
+Well designed:
+- Fills a specific purpose
+- Easy to understand/learn
+- Can be built on/added to
+- Accumulated features/use cases over time
+
+Poorly designed:
+- Small and immutable
+- Large and overly complex
+- Does not fill a niche
+- Tries to do too much
+
```

Comment:
Another key part that you hinted at in the well designed list is that a common symptom of poorly designed programs is to have "master plans" which can make it hard to build off of and can alienate the users.